config/routes.rb

```diff
@@ -1,4 +1,53 @@
 Rails.application.routes.draw do
+  # Routes for the Task doer account:
+
+  # Sign up form
+  get("/task_doer_sign_up", { :controller => "task_doers", :action => "new_registration_form" })
+  
+  # Create record
+  get("/insert_task_doer", { :controller => "task_doers", :action => "create"  })
```

Comment:
The should most likely be a `post` request I think. If an account has a `text` or file column they will need it. Also to not have the password in the form.